src/main/webapp/Portal.html

```diff
@@ -214,7 +217,7 @@
       )
       loadJs(cdnEndpoint + 'js/linkify.min.js')
       loadJs(cdnEndpoint + 'js/linkify-string.min.js')
-      loadCss(cdnEndpoint + 'css/katex-0.10.1.min.css')
```

Comment:
Could [this file](https://github.com/Sage-Bionetworks/SynapseWebClient/blob/develop/src/main/webapp/css/katex-0.10.1.min.css) be removed or is it still used elsewhere?